### ISSUE TITLE
Custom fonts does not work on New arch app

### ISSUE THREAD

**Issue by jihoobyeon** [microsoft/react-native-windows#13662]
### Problem Description

I tried to add a custom font to New arch app - as same method to old arch apps, described at #1254
but the fonts in app doesn't change, even after clean rebuild.

It seems to be folder hierarchy has changed from old arch to new arch, so I think method may also changed, but there aren't any documentations about it.

### Steps To Reproduce

1. Put the font file into the `windows\%projectname%\Assets` folder.
2. Add font into project `%projectname%` in Visual Studio and set properties to Exculde:NO and Distribute:YES.
3. Build the app.

### Expected Results

App's fonts should changed.

### CLI version

13.6.9

### Environment

```markdown
System:
  OS: Windows 10 10.0.19045
  CPU: "(4) x64 AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx  "
  Memory: 22.64 GB / 31.63 GB
Binaries:
  Node:
    version: 20.10.0
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 3.8.4
    path: ~\AppData\Roaming\npm\yarn.CMD
  npm:
    version: 10.8.2
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
  Windows SDK:
    AllowDevelopmentWithoutDevLicense: Enabled
    AllowAllTrustedApps: Enabled
    Versions:
      - 10.0.18362.0
      - 10.0.19041.0
      - 10.0.22000.0
      - 10.0.22621.0
IDEs:
  Android Studio: Not Found
  Visual Studio:
    - 17.11.35222.181 (Visual Studio Community 2022)
Languages:
  Java: 21.0.3
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.3.1
    wanted: 18.3.1
  react-native:
    installed: 0.75.2
    wanted: ^0.75.2
  react-native-windows:
    installed: 0.75.1
    wanted: ^0.75.1
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Target Platform Version

10.0.22621

### Target Device(s)

Desktop

### Visual Studio Version

Visual Studio 2022

### Build Configuration

Debug

**Comment by chrisglein** [comment]
Yep, this is not yet implemented in the new arch. Great callout!

**Comment by jihoobyeon** [comment]
> Yep, this is not yet implemented in the new arch. Great callout!

Oh, then When it will be implemented? Will it take long time?

**Comment by ArksHub** [comment]
Anything?

**Comment by rafaelgomesyupi** [comment]
same

**Comment by anupriya13** [comment]
Track here https://github.com/microsoft/react-native-windows/pull/14495

**Comment by Ferry-200** [comment]
I’ve encountered the same issue. Also in RNW new arch, ver 80

I followed the steps outlined in [docs/SETUP-REACT-NATIVE.md#windows](https://github.com/oblador/react-native-vector-icons/blob/master/docs/SETUP-REACT-NATIVE.md#windows) to move the font files to the specified locations, and in Visual Studio, I changed the properties of the font files to “Exclude: no” and “Distribute: true”. I also added the “font include” and “font filter” settings to the vsproj files, as suggested in the diff provided by [DylanAkp/EmberTune/commit](https://github.com/DylanAkp/EmberTune/commit/27d9263aab915eaf9ecc6627b2ad051e71b5072f).

However, I still haven’t been able to see the icons as expected.
